randovania/data/json_data/prime3.txt

```diff
@@ -1092,26 +1163,41 @@ Hillside Vista
 Asset id: 13902157786380317936
```

Comment:
For Falls of Fire (above), Dock to Hidden Court connection to Dock to G.F.S. Theseus, change requirements to:

All of the Following
Falls of Fire Wall Broken
Ice Missile or Screw Attack